apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -553,8 +557,15 @@ export default Vue.extend({
         this.filterMode === FILTER_MODE_BUILDER &&
         this.filter.type && this.filter.field && this.filter.value
       ) {
-        
-        return [this.filter]
+        if (this.filter.type === 'in') {
+          const vals = this.filter.value.split(/,/)
```

Comment:
I think this should probably include whitespace either side of the comma - `/\s*,\s*/`